twoliter/embedded/Makefile.toml

```diff
@@ -540,6 +541,61 @@ fi
 '''
 ]
 
+# Task to lint Bottlerocket Security Advisories by checking for valid CVE or GHSA IDs
+# and verifying that each versioned directory under "advisories" has a corresponding
+# tag on the Twoliter project this task runs against.
+[tasks.check-advisories]
+script_runner = "bash"
+script = [
+'''
+if find advisories -name '*.toml' -type f >/dev/null 2>&1 ; then
```

Comment:
```suggestion
if find ${BUILDSYS_ROOT_DIR}/advisories -name '*.toml' -type f >/dev/null 2>&1 ; then
```

We should use full filepaths in case the current working directory is not set as expected. There are a few cases in here that refer to the `advisories` directory that require this.